[2021-03-23 12:22:30,584] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: tutorial.data_preperation 2021-03-22T00:00:00+00:00 [queued]>
[2021-03-23 12:22:30,593] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: tutorial.data_preperation 2021-03-22T00:00:00+00:00 [queued]>
[2021-03-23 12:22:30,593] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-23 12:22:30,593] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-03-23 12:22:30,593] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-23 12:22:30,601] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): data_preperation> on 2021-03-22T00:00:00+00:00
[2021-03-23 12:22:30,604] {standard_task_runner.py:52} INFO - Started process 37084 to run task
[2021-03-23 12:22:30,612] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'tutorial', 'data_preperation', '2021-03-22T00:00:00+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', '/Users/adhithyakarthikeyan/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/test_1.py', '--cfg-path', '/var/folders/yw/952dv3rd0332_2c7vqyq34vc0000gp/T/tmpxh2a1xoo', '--error-file', '/var/folders/yw/952dv3rd0332_2c7vqyq34vc0000gp/T/tmpowjxhgo1']
[2021-03-23 12:22:30,615] {standard_task_runner.py:77} INFO - Job 11: Subtask data_preperation
[2021-03-23 12:22:35,658] {logging_mixin.py:104} INFO - Running <TaskInstance: tutorial.data_preperation 2021-03-22T00:00:00+00:00 [running]> on host FACT-LAP-135.local
[2021-03-23 12:22:40,690] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=skadhithya95@example.com
AIRFLOW_CTX_DAG_OWNER=adhithya
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=data_preperation
AIRFLOW_CTX_EXECUTION_DATE=2021-03-22T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-22T00:00:00+00:00
[2021-03-23 12:22:40,691] {logging_mixin.py:104} INFO - 1
[2021-03-23 12:22:40,692] {logging_mixin.py:104} INFO - hello
[2021-03-23 12:22:40,693] {python.py:118} INFO - Done. Returned value was: None
[2021-03-23 12:22:40,699] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=data_preperation, execution_date=20210322T000000, start_date=20210323T065230, end_date=20210323T065240
[2021-03-23 12:22:40,715] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-23 12:22:40,757] {local_task_job.py:146} INFO - Task exited with return code 0
